README.md

```diff
@@ -47,5 +47,48 @@ a `rescue_from` block.
 By default, in development and test mode, a special mock view will be inserted if real credentials are not present. To
 disable this, set the `mock_enable` property of the configuration to false.
 
+## Client Configuration
+
+The `cloudflare_turnstile` view helper simplifies Turnstile implementation by pre-populating the Turnstile tag with `sitekey`, `size`, `action`, and `theme`. However, you can pass your own [data attributes supported by Turnstile](https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/#configurations) to the `cloudflare_turnstile` helper as needed.
+
+Example usage:
+```erb
+<%= cloudflare_turnstile(data: { callback: 'exampleJsCallback', 'error-callback': 'handleErrorJsCallback' }) %>
+```
+
+### Configuration Reference Table
```

Comment:
I dont love the idea of having to keep this table up-to-date with Cloudflare's configuration. Probably best if we leave it as a link to the docs.